packages/pretty-format/src/plugins/HTMLElement.js

```diff
@@ -0,0 +1,104 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @flow
+ */
+
+'use strict';
+
+import type {Colors, Indent, Options, Print, Plugin} from '../types.js';
+
+const escapeHTML = require('./lib/escapeHTML');
+const HTML_ELEMENT_REGEXP = /(HTML\w*?Element)/;
+const test = isHTMLElement;
+
+function isHTMLElement(value: any) {
+  const toStringed = value.toString();
```

Comment:
Yup, I think we could make it even more performant. According to [ElementImpl](https://github.com/tmpvar/jsdom/blob/a6acac4e9dec4f859fff22676fb4e9eaa9139787/lib/jsdom/living/nodes/Element-impl.js#L92) we could check for property `nodeType`. Although this prop alone could possibly result in some false positives (`nodeType` feels not so unique...), it's good for an early check. Only when we know there's such prop, we could perform constructor and regex check:
```js
const test = (value: any) =>
  value !== undefined &&
  value.nodeType === 1 && 
  value.constructor !== undefined &&
  HTML_ELEMENT_REGEXP.test(value.constructor.name);
```